WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -221,17 +223,27 @@ - (void)fetchPostsFromEndpoint:(NSURL *)endpoint
                            failure:(void (^)(NSError *))failure
 {
     NSString *path = [endpoint absoluteString];
-    
     [self.wordPressComRestApi GET:path
            parameters:params
               success:^(id responseObject, NSHTTPURLResponse *httpResponse) {
                   if (!success) {
                       return;
                   }
+
+                  // NOTE: If an offset param was specified sortRank will be derived
+                  // from the offset + order of the results.
+                  BOOL rankByOffset = [params objectForKey:@"offset"] != nil;
+                  __block CGFloat offset = [[params numberForKey:@"offset"] floatValue];
```

Comment:
Not sure if there would be any unexpected results if `rankByOffset` was evaluated as `true` while the actual value for `[[params numberForKey:@"offset"] floatValue]` was `0` as a valid `NSNumber` obj?